perf_test/blas/blas3/KokkosBlas_trtri_perf_test.hpp

```diff
@@ -421,6 +424,9 @@ void __do_trtri_parallel_batched_template(options_t options,
     // Fence after each batch operation
     Kokkos::fence();
   }
+#else
+  timer.reset();
```

Comment:
Why not just skip outputting anything to CSV, if you didn't run the benchmark? Even if the timer is reset, it could still give back some tiny nonzero time.

Reply:
@brian-kelley: I agree. Something like the following should be printed to indicate the test was skipped:
```
  std::cerr
      << std::string(__func__)
      << " disabled since KOKKOS_ENABLE_OPMPTARGET is defined."
      << std::endl;
```